<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetingWindow</class>
 <widget class="QWidget" name="SetingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SetingWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="gb_OtherTools">
     <property name="title">
      <string>第三方工具</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="lb_N2N">
          <property name="text">
           <string>N2N</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>86</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lb_N2NAddr">
          <property name="text">
           <string>未设置</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>86</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="tb_ChoseN2NAddr">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>SSH</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>86</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lb_SSHAddr">
          <property name="text">
           <string>未设置</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>85</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="tb_ChoseSSHAddr">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>FTP</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>86</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lb_FTPAddr">
          <property name="text">
           <string>未设置</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>85</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="tb_ChoseFTPAddr">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>编辑器</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>86</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lb_EditorAddr">
          <property name="text">
           <string>未设置</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>117</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="tb_ChoseEditorAddr">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>ToolforCJQ</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>86</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lb_ToolForCJQAddr">
          <property name="text">
           <string>未设置</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>86</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="tb_ChoseToolForCJQAddr">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QGroupBox" name="gb_Log">
     <property name="title">
      <string>日志</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="cb_OpenLog">
          <property name="text">
           <string>打开</string>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="lb_LogNumTitle">
            <property name="text">
             <string>文件个数</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_LogNum"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="lb_LogSizeTitle">
            <property name="text">
             <string>文件大小</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_LogSize">
            <property name="suffix">
             <string>MB</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="lb_LogLevelTitle">
            <property name="text">
             <string>日志等级</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_LogLevel"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="gb_DataMoniter">
     <property name="title">
      <string>数据监控</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_18">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="lb_DataMonitTimeTitle">
          <property name="text">
           <string>周期S</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_DataMonitTime">
          <property name="suffix">
           <string>S</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QGroupBox" name="gb_Font">
     <property name="title">
      <string>字体(待开发)</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>23</y>
        <width>103</width>
        <height>65</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="lb_FontSizeTitle">
           <property name="text">
            <string>大小</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_FontSize"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="lb_FontTypeTitle">
           <property name="text">
            <string>类型</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_FontType"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <widget class="QGroupBox" name="gb_ProCheck">
     <property name="title">
      <string>程序检测(已删除)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="cb_ProCheckOpen">
          <property name="text">
           <string>打开</string>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="lb_PeoCheckTimeTitle">
            <property name="text">
             <string>周期S</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_PeoCheckTime">
            <property name="suffix">
             <string>S</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QGroupBox" name="gb_OtherSetting">
     <property name="title">
      <string>其他设置</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QCheckBox" name="cb_IsSHHTogetherFTP">
        <property name="text">
         <string>SSH和FTP窗口合并</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QGroupBox" name="gb_N2N">
     <property name="title">
      <string>N2N</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="lb_N2NIP">
          <property name="text">
           <string>IP后缀</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_N2NIP"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>FTP、SSH Jzhs默认目录(必须以/开头、结尾)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_FTPSSHDeflautPath">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QPushButton" name="pb_OpenConfJson">
       <property name="text">
        <string>查看配置文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_Confirm">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <widget class="QGroupBox" name="gb_SoftSkin">
     <property name="title">
      <string>皮肤</string>
     </property>
     <widget class="QComboBox" name="cb_SoftSkin">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
